internal/generator/utils_test.go

```diff
@@ -95,3 +95,26 @@ func Test_checkDefaultValue(t *testing.T) {
 		})
 	}
 }
+
+func Test_normalizeName(t *testing.T) {
+	cases := []struct {
+		t        string
+		val      string
+		expected string
+	}{
+		{t: "int", val: "int", expected: "int"},
+		{t: "*int", val: "*int", expected: "int"},
+		{t: "[]int", val: "int", expected: "int"},
```

Comment:
Here i see different test names (please rename `t` to `name` for simple reading) and same input values `val`.